controller/konnect/index_kongpluginbinding.go

```diff
@@ -11,6 +11,14 @@ const (
 	IndexFieldKongPluginBindingKongPluginReference = "kongPluginRef"
 	// IndexFieldKongPluginBindingKongClusterPluginReference is the index field for KongClusterPlugin -> KongPluginBinding.
 	IndexFieldKongPluginBindingKongClusterPluginReference = "kongClusterPluginRef"
+	// KongServiceRef is the index field for KongService -> KongPluginBinding.
```

Comment:
```suggestion
	// IndexFieldKongPluginBindingKongServiceReference is the index field for KongService -> KongPluginBinding.
```
I think using constant name here (not their values) in comments is better.